build_projects/shared-build-targets-utils/Utils/SharedFrameworkPublisher.cs

```diff
@@ -160,6 +160,14 @@ public void PublishSharedFramework(string outputRootDirectory, string commitHash
             var content = $@"{commitHash}{Environment.NewLine}{version}{Environment.NewLine}";
             File.WriteAllText(Path.Combine(sharedFrameworkNameAndVersionRoot, ".version"), content);
 
+            // Populate symbols publish folder
+            string sharedFrameworkNameAndVersionWithSymbolsRoot = $"{outputRootDirectory}.symbols";
```

Comment:
It is kind of annoying to see the duplication of the path creation here... but I will leave that along until we covert to BuildTools.